src/ARControllerNFT.js

```diff
@@ -105,20 +107,23 @@ export default class ARControllerNFT {
 
     // detect NFT markers
     const nftMarkerCount = this.nftMarkerCount
+    console.log(nftMarkerCount);
     this.detectNFTMarker()
 
     // in ms
     const MARKER_LOST_TIME = 200
 
     for (let i = 0; i < nftMarkerCount; i++) {
       const nftMarkerInfo = this.getNFTMarker(i)
+      console.log(nftMarkerInfo);
```

Comment:
this does not works, We should get the nftMarkerInfo but it don't. Where is the bug?